src/openeo_odc/map_processes_odc.py

```diff
@@ -62,7 +62,7 @@ def exclusive_date(date):
     if 'crs' in process['arguments']['spatial_extent']:
         params['crs'] = process['arguments']['spatial_extent']['crs']
     params['measurements'] = []
-    if 'bands' in process['arguments'] and len(process['arguments']['bands'])>0:
+    if 'bands' in process['arguments'] and not process['arguments']['bands'] is None and len(process['arguments']['bands'])>0:
```

Comment:
It is best practice to write `x is not None` instead of `not x is None`, the latter is just more difficult to read